src/modules/Lan.rb

```diff
@@ -736,7 +662,7 @@ def ProposeVirtualized
     def Packages
       pkgs = []
 
-      if NetworkService.is_network_manager
+      if yast_config&.backend?(:network_manager)
```

Comment:
NP: seems that designing aliases like ```config.network_manager?``` or something like that would make code more pretty.

Reply:
We could defined them, and even do some meta-programming here, defining the methods dynamically. But I know we are not big fun of it... so I would leave it as it is by now.

@imobachgs, what do you think?

Reply:
I am not against but, to be honest, I do not see the benefit.